packages/circuit-ui/components/Modal/Modal.tsx

```diff
@@ -111,7 +110,11 @@ const contentVariantStyles = ({
       }
     `;
   }
-  return null;
+  return css`
+    ${theme.mq.untilKilo} {
+      height: 100vh;
+    }
```

Comment:
The fix is to set this in `variantStyles` only for `variant="immersive"`. This is implicit, I'm also happy to wrap this in an `if(variant === "immersive")`

Reply:
Hmm, the problem I was trying to address here is that there's no default variant. Since all existing modals don't have a variant, this will effectively remain the default behavior anyway. So what does this fix?

Reply:
Ah, immersive modals use `height` while contextual ones use `max-height`, so it doesn't get overridden 🤦🏻‍♂️

Reply:
I was actually just looking into this. There's no explicit default indeed, this would make `immersive` an implicit default. This is a required TS prop anyways, so modals should set it

Reply:
Is what this fixes clear enough? I'm not sure if there's a better way to do this, open to suggestions as always 🙂